tests/unit/test_published_clusters.py

```diff
@@ -17,15 +58,15 @@ def test_published_clusters():
         "externalId": "32b99cab-e01b-41e7-a29d-509165242c6f",
     }
 
-    unified_dataset_json = {
+    _unified_dataset_json = {
         "id": "unify://unified-data/v1/datasets/8",
         "name": "Project_1_unified_dataset",
         "version": "10",
         "relativeId": "datasets/8",
         "externalId": "Project_1_unified_dataset",
     }
 
-    published_clusters_json = {
+    _published_clusters_json = {
```

Comment:
why the underscore before "published"?

Reply:
A lot of the existing tests do that for internal json variables

Reply:
The leading underscore is meaningful: 
https://www.python.org/dev/peps/pep-0008/#method-names-and-instance-variables
"Use one leading underscore only for non-public methods and instance variables."